wp-includes/class-wp-offline-page.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * WP_Offline_Page class.
+ *
+ * @package PWA
+ */
+
+/**
+ * This class manages the Offline Page.
+ */
+class WP_Offline_Page {
+
+	/**
+	 * The option name.
+	 *
+	 * @var string
+	 */
+	const OPTION_NAME = 'page_for_offline';
+
+	/**
+	 * Offline Page post ID.
+	 *
+	 * @var int
+	 */
+	protected $offline_page_id = 0;
+
+	/**
+	 * Array of the static pages.
+	 *
+	 * @var array
+	 */
+	protected $static_pages = array(
+		'page_on_front'              => 0,
+		'page_for_posts'             => 0,
+		'wp_page_for_privacy_policy' => 0,
```

Comment:
Shouldn't this be the same as `self::OPTION_NAME`?

Reply:
@westonruter This one is for the privacy page and not the Offline Page.  I was using it for the following purposes:

1. Target each of their page select dropdowns to exclude the Offline Page from it.
2. Exclude each of them from the offline page's select dropdown.

Reply:
Your right. I totally missed this.